.github/workflows/ci-generators.yml

```diff
@@ -96,6 +96,13 @@ jobs:
           REDIS_URL: redis://localhost:${{job.services.redis.ports[6379]}}
           DATABASE_URL: postgres://postgres:postgres@localhost:5432/postgres_test
 
+      - name: scheduler
+        run: cargo run -- generate scheduler
```

Comment:
i would add some more value for example

```
$ cargo run -- generate job send_emails --cron '* * * * ..'

Added `send_emails` to scheduler jobs
Total 12 jobs in `config/development.yaml` under `scheduler:`.
```

This will **upsert** a schedule record.

You can also experiment with english-to-cron libraries

```
$ cargo run -- generate job send_emails --at 'every sunday at 1pm'

Cron schedule: 0 * * * 1 *
Added `send_emails` to jobs
Total 12 jobs in `config/development.yaml` under `scheduler:`.
```


Notice we generate a `job`
If we say `scheduler` it is the thing that runs `jobs`

Its also a good opportunity to centralize and sharpen our concepts and terminology:

* `task` is a one time, you run it any way you like
* `job` is a repeat on a schedule, run by the `scheduler`
* `worker` is something that performs a `background job` from a queue